includes/user-functions.php

```diff
@@ -556,17 +556,20 @@ function give_get_donor_name_by( $id = 0, $from = 'donation' ) {
 
 		case 'donation':
 
-			$user_info = give_get_payment_meta_user_info( $id );
-			$name = $user_info['first_name'] . ' ' . $user_info['last_name'];
+			$donation_info = new Give_Payment( $id );
+			$first_name    = $donation_info->get_meta( '_give_donor_billing_first_name', true );
+			$last_name     = $donation_info->get_meta( '_give_donor_billing_last_name', true );
 
-		break;
+			$name = $first_name . ' ' . $last_name;
```

Comment:
@mehul0810 Check this commit https://github.com/WordImpress/Give/commit/7acf3385d59ea02ff8550af38ec2c039f30752d9.

Reply:
@ravinderk I've reviewed your commit. I'll make sure to not repeat it. Thanks for pointing me to it.